xchange-stream-kucoin/src/main/java/info/bitrich/xchangestream/kucoin/KucoinStreamingAdapters.java

```diff
@@ -0,0 +1,43 @@
+package info.bitrich.xchangestream.kucoin;
+
+import info.bitrich.xchangestream.kucoin.dto.KucoinOrderEventData;
+import info.bitrich.xchangestream.kucoin.dto.KucoinWebSocketOrderEvent;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.dto.trade.MarketOrder;
+
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+public class KucoinStreamingAdapters {
+    public static Order adaptOrder(KucoinWebSocketOrderEvent orderEvent) {
+        KucoinOrderEventData data = orderEvent.data;
+
+        Order.OrderType orderType = data.side.equals("buy") ? Order.OrderType.BID : Order.OrderType.ASK;
```

Comment:
constant on left side

Reply:
why are you insisting on doing yoda conditions?

most ppl don't find them easier to read https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
It's not about ease of reading it's about preventing an NPE as I've described before.